stable/kong/values.yaml

```diff
@@ -0,0 +1,268 @@
+# Default values for kong.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: kong
+  tag: 0.11.2
+  pullPolicy: IfNotPresent
+
+# Specify Kong admin and proxy services configurations
+admin:
+  # HTTPS traffic on the admin port
+  https:
+    servicePort: 8444
+    containerPort: 8444
+  http:
+    servicePort: 8001
+    containerPort: 8001
+  # Kong admin service type
+  type: NodePort
+  nodePort: 32444
+proxy:
+  # HTTPS traffic on the proxy port
+  https:
+    servicePort: 8443
+    containerPort: 8443
+  http:
+    servicePort: 8000
+    containerPort: 8000
+  type: NodePort
+  nodePort: 32443
+
+# Set runMigrations to run Kong migrations
+runMigrations: true
+
+# Specify Kong configurations
+# Kong configurations guide https://getkong.org/docs/latest/configuration/
+env:
+  database: postgres
+  pg_password: kong
+
+# If you want to specify resources, uncomment the following
+# lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+resources: {}
+  # limits:
+  #  cpu: 100m
+  #  memory: 128Mi
+  # requests:
+  #  cpu: 100m
+  #  memory: 128Mi
+
+# readinessProbe for Kong pods
+readinessProbe:
+  httpGet:
+    path: "/status"
+    port: admin-ssl
+    scheme: HTTPS
+  initialDelaySeconds: 300
+  timeoutSeconds: 60
+  periodSeconds: 60
+  successThreshold: 1
+  failureThreshold: 5
+
+# livenessProbe for Kong pods
+livenessProbe:
+  httpGet:
+    path: "/status"
+    port: admin-ssl
+    scheme: HTTPS
+  initialDelaySeconds: 300
+  timeoutSeconds: 60
+  periodSeconds: 60
+  successThreshold: 1
+  failureThreshold: 5
+
+# Affinity for pod assignment
+# Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+# affinity: {}
+
+# Tolerations for pod assignment
+# Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+tolerations: []
+
+# Node labels for pod assignment
+# Ref: https://kubernetes.io/docs/user-guide/node-selection/
+nodeSelector: {}
+
+# Annotation to be added to Kong pods
+podAnnotations: {}
+
+# Kong pod count
+replicaCount: 1
+
+
+# Kong has a choice of either Postgres or Cassandra as a backend datatstore.
+# This chart allows you to choose either of them with the `database.type`
+# parameter.  Postgres is chosen by default.
+
+# Additionally, this chart allows you to use your own database or spin up a new
+# instance by using the `postgres.enabled` or `cassandra.enabled` parameters.
+# Enabling both will create both databases in your cluster, but only one
+# will be used by Kong based on the `env.database` parameter.
+# Postgres is enabled by default.
+
+# Cassandra chart configs
```

Comment:
It doesn't make sense to reproduce the whole Cassandra and Postgres values files here.